Readme.md

```diff
@@ -0,0 +1,4 @@
+# thala-sdk
+
+- [@thalalabs/thalaswap-math](./packages/thalaswap-math/)
+- [@thalalabs/router-sdk](./packages/thalaswap-router/)
```

Comment:
can we name router-sdk to thalaswap-router? not sure how easy is it to rename a npm package

Reply:
@0xbe1 it will simply release a new package

Reply:
anyway, merge it now. We can use new name to publish the package if we want.